10-pattern/01-negation.md

```diff
@@ -815,6 +818,7 @@ rule unreachability-rule:
 ```java
 GraqlDefine query = Graql.define(
     type("unreachable").sub("relation").relates("from").relates("to"),
+    type("traversable").sub("entity").plays("unreachable", "from").plays("unreachable", "from"),
```

Comment:
should the 2nd one be `.plays("unreachable", "to")`?

Reply:
Fixed in https://github.com/graknlabs/docs/pull/489/commits/3d0769fb7ccfc3128ff90cac41fb010e7acf5099